package.json

```diff
@@ -18,39 +18,46 @@
     "url": "https://github.com/bitmovin/bitmovin-player-ui/issues"
   },
   "homepage": "https://github.com/bitmovin/bitmovin-player-ui#readme",
+  "browserslist": [
+    "> 1%",
+    "last 2 versions",
+    "Firefox ESR"
+  ],
   "devDependencies": {
+    "@inrupt/jest-jsdom-polyfills": "^1.4.3",
     "@types/jest": "^24.0.11",
     "@types/jsdom": "^12.2.4",
-    "autoprefixer": "^6.5.2",
+    "autoprefixer": "^10.4.13",
     "bitmovin-player": "^8.28.0",
     "browser-sync": "^2.26.3",
     "browserify": "^13.1.1",
-    "cssnano": "^3.8.0",
+    "cssnano": "^5.1.14",
     "del": "^2.2.2",
     "gulp": "^4.0.0",
     "gulp-css-base64": "^1.3.4",
     "gulp-header": "^2.0.5",
-    "gulp-postcss": "^6.2.0",
+    "gulp-postcss": "^9.0.1",
     "gulp-rename": "^1.4.0",
     "gulp-replace": "^0.6.1",
     "gulp-sass": "^5.1.0",
     "gulp-sass-lint": "^1.4.0",
     "gulp-sourcemaps": "^2.6.4",
     "gulp-tslint": "^8.1.3",
-    "gulp-typescript": "^3.2.4",
+    "gulp-typescript": "^6.0.0-alpha.1",
```

Comment:
Oof, should we really use an `alpha` version here? 😅

Reply:
Dependabot thought it's a great idea. Who am I to not trust it? 😅

Reply:
`6.0.0-alpha.1` was published 3 years ago and is the last publish of `gulp-typescript` as it seems: https://www.npmjs.com/package/gulp-typescript
I can try downgrading to 5.x if you feel safer with that?

Reply:
Done in 7f709a9b

Reply:
dependabot is not always to be trusted 😅 
yeah if it doesn't make a difference, I'd stay with a stable release :)

Reply:
Apparently 😅 
Agreed, downgraded to the latest stable 5.x